src/main/java/com/zufar/onlinestore/payment/dto/CreatePaymentMethodDto.java

```diff
@@ -0,0 +1,18 @@
+package com.zufar.onlinestore.payment.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
+public record CreatePaymentMethodDto (
+
+        @NotBlank(message = "cardNumber is the mandatory attribute")
+        String cardNumber,
+
+        @NotBlank(message = "expMonth is the mandatory attribute")
+        Long expMonth,
+
+        @NotBlank(message = "expYear is the mandatory attribute")
+        Long expYear,
+
+        @NotNull(message = "cvc is the mandatory attribute")
+        String cvc) {}
```

Comment:
please, move the curly bracket to the new line
```
  ...
  @NotNull(message = "cvc is the mandatory attribute")
   String cvc) {
  }
```

Reply:
Done